smart/map_processing.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import skimage as ski
+from matplotlib import colors
+from skimage.morphology import disk
+
+from sunpy.map import Map, all_coordinates_from_map, coordinate_is_on_solar_disk
+
+
+def map_processing(file):
+    """
+    Creating a map from a fits file and processing said map.
+
+    Parameters
+    ----------
+    file : fits
+        fits file to turn into map.
+
+    Returns
+    -------
+    im_map : sunpy.map.Map
+        processed map created from fits file.
+
+    """
+    im_map = Map(file)
+    if im_map.meta["CROTA2"] >= 100:
+        data = np.flip(im_map.data, 1)[::-1]
+        im_map = Map(data, im_map.meta)
```

Comment:
What is this doing? If reprojecting from one map to another shouldn't need, if do need `map.rotate` might be better